examples/tutorial/lessons/lesson-03-dampening.adoc

```diff
@@ -0,0 +1,202 @@
+
+=== Condition Evaluations
+
+Note that an _evaluation_ happens whenever we have new incoming data that allows us to evaluate the condition set.  Triggers can define _ALL_ or _ANY_ matching to the condition set.  _ALL_ is the default. It means that using the most recent data available for each condition, each condition must individually evaluate to true for the condition set to evaluate to true, as a whole. _ANY_ matching is what you would expect, only one of multiple conditions must evaluate to true for the condition set to be true.
+
+Dampening is concerned with *condition set* evaluations, not individual conditions.
+
+=== Relaxed Dampening
+
+Sometimes strict dampening is, well, too strict.  In that case we have relaxed dampening, again in the count and time varieties.  Relaxed-count means N out of M consecutive true evaluations.  Relaxed-time means N true evaluations in some minimum period of time.  One note on relaxed time, unless data is reported frequently enough the dampening may never be satisfied.  To varying degrees the time based dampening options are tied to the reporting intervals of the relevant dataIds.
+
+You can find http://www.hawkular.org/community/docs/developer-guide/alerts.html#_trigger_dampening[more details on dampening options here].
+
+=== Example
+
+Now, let's add to the trigger example we started in lesson 02.
+
+We'll do two things:
+
+* To show a multi-condition trigger let's add a _Rate_ condition on our counter-1 metric.
+** Remember, this metric increases by 0..5 every 5s, which extrapolates to 0..60 every minute.  We'll look for GTE 30 per minute.
+* Add Strict-2 dampening.
+** That means the condition set (both conditions) must be true on two consecutive evaluations.
+** Remember, when data arrives at different times, the most recent data for each condition is used for the evaluations.
+
+In case you didn't just run lesson 02, we'll start with a clean slate.
+
+. If necessary, revisit lesson 01.  Restart your docker-compose via:
+.. `> sudo docker-compose down`
+.. `> sudo docker-compose up`
+. If your data pump script, tutorial-data.sh is running, stop it.
+
+  
+Let's create our trigger introducing a new endpoint, _Full trigger_ endpoints are convenient for dealing with complete trigger definitions. We can create, fetch, import and export full triggers.  Here, we'll use a POST to create the full trigger:
+
+[cols="1,5l"]
+|===
+|Request
+|POST http://localhost:8080/hawkular/alerts/triggers/trigger
+
+|HTTP Headers
+|Hawkular-Tenant: tutorial
+Content-Type: application/json
+
+|Body
+|
+{
+"trigger": {
+  "tenantId": "tutorial",
+  "id": "lesson-03",
+  "name": "Lesson 03",
+  "description": "A useful description that can also include resolution instructions or links",
+  "autoDisable": true,
+  "enabled": true,
+  "firingMatch": "ALL"
+},
+"conditions": [
+  {
+    "tenantId": "tutorial",
```

Comment:
When using a full trigger there is no need to repeat the tenantId and triggerId on conditions and dampenings.
I think that is more clear and reduces the redundant fields.
As example:
https://github.com/lucasponce/hawkular-alerts-demo/blob/master/03_import-definitions/import-applications.json

Also, perhaps FIRING is not necessary as it is the default one.